server/clients/jsonApiClient.js

```diff
@@ -0,0 +1,35 @@
+const axios = require('axios');
+const httpAdapter = require('axios/lib/adapters/http');
+const { logger } = require('../utils/logger');
+
+class JsonApiClient {
+  constructor(baseURL) {
```

Comment:
All the other base clients we have require the verb methods to specify the url. This client only requires the path specifying, so:  

`jsonApi.get('/some-path')`

vs

`jsonApi.get('http:some-site/some-path')`